docs/api/cpp/roundingmode.rst

```diff
@@ -1,10 +1,10 @@
 RoundingMode
 ============
 
-.. doxygenenum:: cvc5::api::RoundingMode
+.. doxygenenum:: cvc5::RoundingMode
```

Comment:
This generates a warning (cannot find enum "cvc5::RoundingMode"): https://cvc5.github.io/docs-ci/docs-pr8455/api/cpp/roundingmode.html

Reply:
I think that's due to https://github.com/cvc5/cvc5/pull/8427, I'll fix it in this PR.